v5/api/c/imu.rst

```diff
@@ -15,7 +15,7 @@ imu_reset
 
 Calibrate IMU.
 
-This takes approximately 2 seconds, and is a non-blocking operation.
+This takes approximately 2 seconds, and is blocking until the IMU status flag is set properly.
```

Comment:
I'd just say until the Imu is initialized. I'm not sure what the status flag is.

Reply:
file:///C:/Users/DELL-E~1/AppData/Local/Temp/Rar$EXa16940.23492/v5/api/c/imu.html?highlight=imu#imu-get-status , it's related to this status flag (I'll change it to state to be slightly more accurate though).

Reply:
It's related to imu_get_status, I guess I could change it to "until IMU status is set properly".